recipes/twitchtv-libsoundtrackutil/all/test_package/example.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright Twitch Interactive, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+#include <libsoundtrackutil/SoundtrackIPC.h>
```

Comment:
Since we are creating a test package without their knowledge let's not tangle them into this =)

```suggestion
#include <libsoundtrackutil/SoundtrackIPC.h>
```